providers/implementations/rands/drbg.c

```diff
@@ -365,9 +365,6 @@ int ossl_prov_drbg_instantiate(PROV_DRBG *drbg, unsigned int strength,
     size_t noncelen = 0, entropylen = 0;
     size_t min_entropy, min_entropylen, max_entropylen;
 
-    if (!ossl_prov_is_running())
```

Comment:
Why was this removed??

Reply:
The one in rand_drbg_restart() is ok.. So as long as the instatiate wrapper ones do the check then this is ok,.

Reply:
I added this check to the three DRBGs along with the set_params call.

I contemplated leaving it there to avoid this question it but figured it was better to do the job properly 😃

Reply:
Maybe just keep in mind if this code 'changes'.